### PR TITLE
fixing labels in case of SAML

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1938

### CODE REVIEW

**Review by jenny-s51 — CHANGES_REQUESTED** [review]

File: src/identity-providers/add/SamlConnectSettings.tsx [hunk 1]
```diff
@@ -70,13 +71,18 @@ export const SamlConnectSettings = () => {
...
       >
         <TextInput
           type="text"
           name="config.entityId"
           data-testid="serviceProviderEntityId"
           id="kc-service-provider-entity-id"
-          ref={register()}
+          ref={register({ required: true })}
+          validated={errors.config?.entityId ? "error" : "default"}
+          placeholder={`${environment.authServerUrl}/auth/realms/${realm}`}
```
[the remark this answers is not in the document]
Comment (edewit) [reply src/identity-providers/add/SamlConnectSettings.tsx]: I think it's an example of what the value could be like, not the actual value

**Comment by edewit** [comment]
On cypress on this PR it passed

**Comment by jonkoops** [comment]
Running them locally seems to pass them as well now. I'll run the full test suite, we can merge if it's green.

**Comment by jonkoops** [comment]
Looks like all tests are passing now! :heavy_check_mark: 
```log
  (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  authentication_policies.spec.ts          00:14        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  authentication_test.spec.ts              01:10       15       15        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  client_authorization_test.spec.ts        00:22        9        9        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  client_scopes_test.spec.ts               00:19        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  clients_saml_test.spec.ts                00:15        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  clients_test.spec.ts                     00:52       15       15        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  events_test.spec.ts                      00:49       14       14        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  group_test.spec.ts                       00:35        9        9        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  identity_providers_oidc_test.spec.t      00:26        5        5        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  identity_providers_saml_test.spec.t      01:02       12       12        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  identity_providers_test.spec.ts          00:42        9        9        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  login_test.spec.ts                       00:06        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  masthead_test.spec.ts                    00:14        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  partial_export_test.spec.ts              00:12        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  partial_import_test.spec.ts              00:36        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_roles_test.spec.ts                 00:22        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_client_policies_test      00:46       18       18        -        -        - │
  │    .spec.ts                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_client_profiles_test      00:44       21       21        -        -        - │
  │    .spec.ts                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_events_test.spec.ts       01:27       17       16        -        1        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_tabs_test.spec.ts         00:25        6        6        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_user_profile_tab.spe      00:13        3        3        -        -        - │
  │    c.ts                                                                                        │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_test.spec.ts                       00:10        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_user_registration.spec.ts          00:08        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  sessions_test.spec.ts                    00:35       11       11        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_fed_kerberos_test.spec.ts           00:45        9        9        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_fed_ldap_hardcoded_mapper_test      01:18       12       12        -        -        - │
  │    .spec.ts                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_fed_ldap_mapper_test.spec.ts        01:22       12       12        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_fed_ldap_test.spec.ts               00:50        8        8        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  users_test.spec.ts                       00:27       13       12        -        1        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        17:40      251      249        -        2        -  
```

**Review by jonkoops — APPROVED** [review]

**Pull request merged**